opentracing_instrumentation/request_context.py

```diff
@@ -23,11 +23,15 @@
 import threading
 
 import opentracing
-import tornado.stack_context
+from opentracing.scope_managers.tornado import TornadoScopeManager
```

Comment:
are we sure that TornadoScopeManager is thread-safe? Specifically, see #71

Reply:
It should be, based on https://github.com/opentracing/opentracing-python/blob/master/opentracing/scope_managers/tornado.py#L154 - in any case, will double check by merging your latest test case and try things out ;)